src/pip/_internal/cli/cmdoptions.py

```diff
@@ -235,9 +235,9 @@ class PipOption(Option):
     "--progress-bar",
     dest="progress_bar",
     type="choice",
-    choices=["on", "off"],
-    default="on",
-    help="Specify whether the progress bar should be used [on, off] (default: on)",
+    choices=["auto", "on", "off"],
+    default="auto",
+    help="Specify whether the progress bar should be used [on, off] (default: auto)",
```

Comment:
We should probably mention what auto means

Reply:
auto=on
on = force on?